src/test/java/core/basesyntax/RegistationServiceTest.java

```diff
@@ -0,0 +1,177 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistationServiceTest {
+    private static RegistrationService registerServiceTest;
+    private static List<User> users = initCorrectUsersList();
+
+    @BeforeAll
+    static void createService() {
+        registerServiceTest = new RegistrationServiceImpl();
+        for (User user : users) {
+            registerServiceTest.register(user);
+        }
+    }
+
+    @Test
+    void register_isNullUser_NotOk() {
+        User nullUser = null;
+        assertThrows(NullPointerException.class, () -> {
+            registerServiceTest.register(nullUser);
+        });
+    }
+
+    @Test
+    void register_isNullLogin_NotOk() {
+        User actual = new User(null, "655234322", 21);
+        try {
+            registerServiceTest.register(actual);
+        } catch (ValidationException e) {
+            return;
+        }
+        fail("ValidatorException should be thrown if login is null");
+    }
+
+    @Test
+    void register_isNullPassword_NotOk() {

```

Comment:
The word after the underscore should start from a lowercase letter 
Fix for all tests 
```suggestion
    void register_isNullPassword_notOk() {

```